ARTICLE-3.markdown

```diff
@@ -61,7 +61,9 @@ the Guiding Council shall have the option of taking on the decision, though they
 must offer the membership an apology upon taking action, which shall include an
 explanation of the need for proceeding without member ratification and steps
 taken to ensure any demonstrated concerns of membership were taken into 
-account. This stipulation does not apply to amendments or changes to these 
+account. 
+
+This stipulation **DOES NOT** apply to amendments or changes to these 
```

Comment:
The only change here is to highlight that if board has the option of going with "Condorcet Vote" if consensus is not reached in all business matters except ratifying membership of a new candidate.